djangocms_blog/search_indexes.py

```diff
@@ -47,32 +44,36 @@ def get_model(self):
         return Post
 
     def get_search_data(self, post, language, request):
-        optional_attributes = []
-        abstract = post.safe_translation_getter('abstract')
-        text_bits = [post.get_title()]
```

Comment:
I'll keep this one, because it returns the meta_title or the title as a fallback, so it's not a real repetition.
See: https://github.com/nephila/djangocms-blog/blob/develop/djangocms_blog/models.py#L228

Btw, we can discuss about it :)!